data/formats-data.js

```diff
@@ -2928,7 +2928,7 @@ exports.BattleFormatsData = {
 		tier: "RU"
 	},
 	duskull: {
-		randomBattleMoves: ["willowisp","shadowsneak","icebeam","painsplit","substitute","nightshade"],
+		randomBattleMoves: ["willowisp","shadowsneak","icebeam","painsplit","substitute","nightshade","toxic","trickroom"],
```

Comment:
Remove Ice Beam. It's way too weak with 30 SpA. I don't like Toxic either. Perhaps Destiny Bond instead (will be quite useful on a Trick Room set).